arch/arm/core/irq_manage.c

```diff
@@ -84,19 +84,18 @@ int _arch_irq_is_enabled(unsigned int irq)
  */
 void _irq_priority_set(unsigned int irq, unsigned int prio, u32_t flags)
 {
-	/* Hardware priority levels 0 and 1 reserved for Kernel use.
```

Comment:
Is it not always true that the kernel reserves priority levels 0 and 1?

Reply:
I can only see that priority level 0 is reserved for SVCs and FAULTs.

Reply:
Yes, this comment is outdated, see my reply below.